Source/WebKit/UIProcess/API/APIContentRuleList.cpp

```diff
@@ -77,6 +78,12 @@ bool ContentRuleList::supportsRegularExpression(const WTF::String& regex)
     return false;
 }
 
+std::error_code ContentRuleList::parseRuleList(const WTF::String& ruleList)
+{
+    Expected<Vector<WebCore::ContentExtensions::ContentExtensionRule>, std::error_code> result = WebCore::ContentExtensions::parseRuleList(ruleList);
```

Comment:
auto

Reply:
> This needs a unit test that causes a parse failure and a unit test that does not.

Yes it does. I'll add those in the next go of this PR.